tests/PHPStan/Analyser/data/instanceof.php

```diff
@@ -134,10 +135,18 @@ public function testExprInstanceof($subject, string $classString, $union, $inter
 
 		if ($subject instanceof $objectTString) {
 			assertType('ObjectT of InstanceOfNamespace\BarInterface (method InstanceOfNamespace\Foo::testExprInstanceof(), argument)', $subject);
-			assertType('true', $subject instanceof $objectT);
+			assertType('true', $subject instanceof $objectTString);
```

Comment:
What about `$objectT`? Shouldn't this be a new assert instead of changing the existing one?

Reply:
The original assert was wrong, it should have been `$objectTString`

Reply:
OK, so the original assert type should be updated :) To `false` probably.